solutions/pandas_tests/Makefile

```diff
@@ -0,0 +1,60 @@
+# Some tests in files listed in tests.partially_passed fail/ error out on execution.
+# These failures and error are caused by one of below:
+# 1. pypaper linux specific errors, documented
+# here - https://pyperclip.readthedocs.io/en/latest/#not-implemented-error
+# 2. s3fs error (which is currently an optional file handling mechanism in pandas)
+# https://bleepcoder.com/evalml/557744402/unit-test-fails-after-upgrading-to-pandas-1-0-0
+#
+# NOTE: "make run-one" is currently not supported as running pytest against a single file
+# could fail unexpectedly (e.g., pandas/tests/extension/arrow/arrays.py fails with
+# ModuleNotFoundError).
+
+TOP = $(abspath ../..)
+include $(TOP)/defs.mak
+
+APPBUILDER=$(TOP)/scripts/appbuilder
+PYTHON3 = /usr/local/bin/python3
+PYTEST_CMD = $(PYTHON3) -m pytest
+PYTEST_INI_FILE = pytest.ini
+PYTEST_OPTS = -s -q --no-header --skip-slow --skip-db --skip-network
+
+OPTS = --max-affinity-cpus=4 --app-config-path config.json
+
+ifdef STRACE
+OPTS += --strace
+endif
+
+all: rootfs
+
+appdir:
+	$(APPBUILDER) Dockerfile
+	cp $(PYTEST_INI_FILE) appdir/
+
+rootfs: appdir
+	$(MYST) mkext2 appdir rootfs
+
+rootfs-passed: appdir
+	# remove test files that are known to fail partially. These are tested in target 'run'
+	# for expected failures/ errors.
+	cat tests.partially_passed | while read line; do rm -f appdir/"$$line"; done
+	$(MYST) mkext2 appdir rootfs-passed
+
+run-passed: rootfs-passed
+	$(MYST_EXEC) rootfs-passed $(OPTS) $(PYTHON3) /app/app.py
+
+run: rootfs
```

Comment:
Can we add a "make one" to run  a single test. `/usr/local/bin/python3 -m pytest /usr/local/lib/python3.9/site-packages/pandas/tests/arrays/string_/test_string.py` worked for me.

Reply:
According to @RRathna, tests could fail with make one but pass with run all. The other way around is supporting make one but documenting how many tests are failing

Reply:
Maybe certain test case relies on state generated by a previous test case. But the suggestion here is to provide a "make one" mechanism to support debugging, not to replace current way to run the full test suite with a "make one" loop over a full list of the test cases.

Reply:
Sure. we can support make one and put current comments there indicates it's for debugging purposes.